odoo/addons/test_http/tests/test_session.py

```diff
@@ -213,3 +217,40 @@ def test_session7_serializable(self):
             with self.assertRaises(TypeError):
                 dict.update(session, foo=value)
             self.assertFalse(session.foo)
+
+
+class TestSessionStore(HttpCaseWithUserDemo):
+    def setUp(self):
+        super().setUp()
+        tmpdir = TemporaryDirectory()
+        config["data_dir"] = tmpdir.name
+        self.addCleanup(tmpdir.cleanup)
+
+    @mute_logger('odoo.http')
+    def test_session_lifetime(self):
+        # default lifetime is 1 week
+        self.env['ir.config_parameter'].set_param('sessions.max_inactivity_seconds', False)
+        with freeze_time() as freeze:
+            auth = self.authenticate(None, None)
```

Comment:
it is a session object that this function returns
```suggestion
            session = self.authenticate(None, None)
```